src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -265,6 +274,15 @@ void setSSLParameters(SSLParameters params) {
             this.signatureSchemes = ss;
         }   // Otherwise, use the default values
 
+        String[] ngs = params.getNamedGroups();
+        if (ngs != null) {
+            // Note if 'ss' is empty, then no signature schemes should be
```

Comment:
The comment needs to be updated for named groups. It looks like it was copied from line 272.

Reply:
Thanks for the catch.  Updated.